sass/directives/_flyout.scss

```diff
@@ -87,3 +87,117 @@
     }
   }
 }
+
+// Use the mobile-subnav mixin for the subnav bars that appear on our subscription / product pages
+// Displays a single link with a flyout
+// Structure of markup should be:
+@mixin mobile-subnav-flyout() {
+  width: auto; 
+  display: none;
+  position: relative;
+  cursor: pointer;
+
+  @media only screen and (max-width: $small-screen-max) {
+    line-height: 45px;
+    display: block;
+    height: $product_subnav--height;
+    margin-left: -$container-edge-padding;
```

Comment:
I think you can get rid of the negative margin with a little refactoring

![image](https://user-images.githubusercontent.com/2359538/27444133-f87eaf4a-573a-11e7-86a6-06ea5904f10b.png)

I think it will make the haml a little easier to read and we'll have a little less CSS to keep up with